misc/package/Data Files/MWSE/core/mcm/components/templates/Template.lua

```diff
@@ -354,4 +354,13 @@ function Template.__index(tbl, key)
 	return Template[key]
 end
 
+--- This will recursively go through your MCM and append the text "Default = ___" to the description of each setting.
+---@param defaultConfig table? the default config of your mod. if not provided, it will try to be retrieved, 
+-- using the path "config.default"
+function Template:addDefaultsToDescriptions(defaultConfig)
+	for _, subComp in ipairs(self.pages) do
+		subComp:addDefaultsToDescriptions(defaultConfig)
+	end
+end
+
```

Comment:
Can you instead of making this a method in `Template`, be behind a flag on `Template:new` method, and if it's set to `true` call the `addDefaultsToDescriptions` method on all of its pages and also call that method in this snippet:
```lua
	Template[key] = function(self, data)
		if not data then
			data = {}
		elseif type(data) == "string" then
			data = { label = data }
		end
		data.parentComponent = self
		local component = mwse.mcm[key](data)
		table.insert(self.pages, component)
		return component
	end
```